src/VisualStudio/Core/Def/Implementation/PreviewPane/PreviewPane.xaml.cs

```diff
@@ -121,7 +121,7 @@ private FrameworkElement CreatePreviewElement(IReadOnlyList<object> previewItems
                 }
                 else if (previewItem is string)
                 {
-                    previewElement = GetPreviewForString((string)previewItem, createBorder: previewItems.Count == 0);
```

Comment:
Note that "previewItems.Count == 0" would always be false.  We could only get in here if previewItems.Count was greater than 0 (since that's the collection we're iterating over).

I removed this code entirely and verified that things look good.